homeassistant/components/water_heater/strings.json

```diff
@@ -4,5 +4,16 @@
       "turn_on": "Turn on {entity_name}",
       "turn_off": "Turn off {entity_name}"
     }
+  },
+  "state": {
+    "_": {
+      "off": "[%key:common::state::off%]",
```

Comment:
Do we need `"on"` as well? Two platforms seem to use that state.

Reply:
I don't think that 'on' should be a supported state. It is not supported according to the [dev docs](https://developers.home-assistant.io/docs/core/entity/water-heater/#states) and in the climate implementation they don't support it as well.

I could add it to this PR in the mean-time, until we implement the improvements for water_heater as discuced here: https://github.com/home-assistant/architecture/discussions/518.

Reply:
If we're not going to support it, let's not add it.